packages/aws-cdk-lib/aws-opensearchservice/lib/domain.ts

```diff
@@ -687,6 +687,15 @@ export interface DomainProps {
    * @default - false
    */
   readonly suppressLogsResourcePolicy?: boolean;
+
+  /**
+   * Whether to enable or disable cold storage on the domain. You must enable UltraWarm storage to enable cold storage.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-opensearchservice-domain-coldstorageoptions.html
```

Comment:
```suggestion
   * @see https://docs.aws.amazon.com/opensearch-service/latest/developerguide/cold-storage.html
```
Let's provide the link to the dev guide instead.